src/kernel/mod.rs

```diff
@@ -23,6 +25,7 @@ impl Context {
         Context {
             vga: Mutex::new(Vga::new(slice)),
             idt: IdtRef::new(),
+            keyboard: Mutex::new(Keyboard::new()),
```

Comment:
Leaving a note for myself that I have some feels here; I'd like to handle this more like `IdtRef`. Will elaborate later.